src/Time.hpp

```diff
@@ -55,9 +56,16 @@ namespace base
 	/** Convert time into a string
          * \param resolution Resolution which the string should present
          * \param mainFormat Main format to use -- this is passed to strftime and appended by ':' plus the
-         *     below seconds resolution if requested by the resolution argument
+         *     below seconds resolution if requested by the resolution argument,
+         *     and additionally suffixed by '%z' (see documentation of strftime)
+         *     the current timezone's offset to UTC
          **/
-	std::string toString(base::Time::Resolution resolution = Microseconds, const std::string& mainFormat = "%Y%m%d-%H:%M:%S") const;
+	std::string toString(base::Time::Resolution resolution = Microseconds,
+                const std::string& mainFormat = base::Time::DEFAULT_FORMAT) const;
```

Comment:
This whole block's indentation is broken on GH's PR view, I'm guessing there are tabs.